physics/speed_of_sound.py

```diff
@@ -0,0 +1,56 @@
+"""
+Title : Calculating the speed of sound
+
+Description :
+    The speed of sound (c) is the speed that a sound wave travels
+    per unit time (m/s). During propagation, the sound wave propagates
+    through an elastic medium. Its SI unit is meter per second (m/s).
+
+    Only longitudinal waves can propagate in liquids and gas other then
+    solid where they also travel in  transverse wave. The following Algo-
+    rithem calculates the speed of sound in fluid depanding on the bulk
+    module and the density of the fluid.
+
+    Equation for calculating speed od sound in fluid:
+    c_fluid = (K_s*p)**0.5
+
+    c_fluid: speed of sound in fluid
+    K_s: isentropic bulk modulus
+    p: density of fluid
+
+
+
+Source : https://en.wikipedia.org/wiki/Speed_of_sound
+"""
+
+
+class SpeedOfSound:
+    @staticmethod
+    def fluid(fluid_density: float, bulk_modulus: float) -> float:
```

Comment:
We don't really use the class so it seems like unnecessary overhead.
```suggestion
def speed_of_sound_in_a_fluid(density: float, bulk_modulus: float) -> float:
```